R/f7Fab.R

```diff
@@ -14,7 +14,8 @@
 #' @param morphTarget CSS selector of the morph target: \code{".toolbar"} for instance.
 #'
 #' @note The background color might be an issue depending on the parent container. Consider
-#' it experimental.
+#' it experimental. Also note that the f7Fabs looses its sticky behaviour when used inside
+#' \link{f7Tabs} and the tabs are animated or swipeable.
```

Comment:
@DivadNojnarg `position:fixed`, makes sure the FABs are sticky in a single layout, but in tab layout this property doesn't do anything when `f7Tabs` has swipeable or animated enabled. This is because in that case, the parent container has the `transform` property set. So to be honest this is a half-baked solution which I'm not super stoked about. Alternatives:

- Move fab container after the navbar container, above .page-content container. Works in both layouts. The way it is designed by Framework7. Downside: if in tab layout, the FABs will be on every tab. This would require some JS to determine which tab the FABs were called from, and hide/show the FABs on tab change.
- Add a scroll listener JS side and make sure the position dynamically changes on scroll. 

From those two options, the first would be my proffered, because that's at least how it was intended. What are your thoughts? And, also important, is it worth it?